base/sysinfo.jl

```diff
@@ -165,6 +165,18 @@ mutable struct UV_cpu_info_t
     cpu_times!idle::UInt64
     cpu_times!irq::UInt64
 end
+"""
+    Sys.CPUinfo
+
+The `CPUinfo` type is a mutable struct with the following fields:
+- `model::String`: CPU model information.
+- `speed::Int32`: CPU speed.
+- `cpu_times!user::UInt64`: Time spent in user mode. CPU state shows CPU time used by user space processes.
```

Comment:
These should document the units.  Is it time in seconds?   And speed is in MHz, I think?

Reply:
yes time is in seconds